parlai/tasks/airdialogue/build.py

```diff
@@ -31,4 +32,14 @@ def build(opt):
         for downloadable_file in RESOURCES:
             downloadable_file.download_file(dpath)
 
+        # Re-organize the directory to be less redundant
+        print('reorganizing airdialogue directory')
```

Comment:
would it make more sense to just download the file to `opt['datapath']` so it unzips properly?

Reply:
Good point did not think of that! I'll fix that